/*
 * This file was automatically generated by EvoSuite
 * Mon Jun 01 14:59:03 GMT 2020
 */

package com.capitalone.dashboard.event.sync;

import org.junit.Test;
import static org.junit.Assert.*;
import com.capitalone.dashboard.event.sync.SyncException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class SyncException_ESTest extends SyncException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      SyncException syncException0 = new SyncException("%mtM;DoH~6{5*`");
  }
}
